pkg/cable/libreswan/libreswan.go

```diff
@@ -131,6 +139,21 @@ func (i *libreswan) Init() error {
 		return fmt.Errorf("error starting Pluto: %v", err)
 	}
 
+	i.ipt, err = iptables.New()
+	if err != nil {
+		return fmt.Errorf("error initializing iptables: %v", err)
+	}
+
+	if err := util.CreateChainIfNotExists(i.ipt, mangleTable, subMarinerPostRoutingChain); err != nil {
+		klog.Errorf("error creating iptables chain %s: %v", subMarinerPostRoutingChain, err)
```

Comment:
It seems like we should propagate this error. Same on L154.

Reply:
Done.